nagios/nagios-metrics-wf.py

```diff
@@ -112,35 +113,75 @@ def remove_trailing_dots(text):
     parser.add_argument('--service', dest='service', action='store_true',
                         help='proccess "service-perfdata" (default: "host-perfdata")')
     parser.add_argument('--test', dest='test', action='store_true')
-    parser.add_argument('--wf_server', dest="server")
-    parser.add_argument('--wf_token', dest="token")
-    parser.add_argument('--wf_proxy_addr', dest="addr")
-    parser.add_argument('--wf_proxy_port', dest="port")
+    parser.add_argument('--wf_server', dest="server", help='Wavefront Server URL.')
+    parser.add_argument('--wf_token', dest="token", help='Wavefront token.')
+    parser.add_argument('--wf_proxy_addr', dest="addr", help='Wavefront Proxy address.')
+    parser.add_argument('--wf_proxy_port', dest="port", help='Wavefront Proxy port.')
+    parser.add_argument('--csp_base_url', dest= "csp_base_url", help='Base URL of CSP console.')
```

Comment:
help='Base URL of the VMware Cloud Services Console.'

Can we provide the actual URL if it's static?

Reply:
As the URL varies for staging and production, I did not hardcode the actual URL.
@gangadharaswamy Please share your thoughts

Reply:
@svinothini24 
Let's use `https://console.cloud.vmware.com` for the CSP Base URL. This is for production clusters.
For internal use, we can modify the url.

Reply:
can we remove this parameter `--csp_base_url` itself for the user?

Reply:
In order to be flexible with the csp_base_url, as that impacts the changes on cfg also, we will get the base url from the command line itself than static